### PR TITLE
Settings activity from template integrated into app.

### PR DESCRIPTION
**Pull request by BarAgam**
Settings activity from template integrated into app.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -45,11 +45,18 @@
...
+
     <activity
         android:name=".app.map.view.DrawActionActivity"
         android:configChanges="orientation|keyboardHidden|screenSize"
         android:label="@string/activity_action_draw_title"
         android:windowSoftInputMode="adjustResize"/>
+
+    <activity
+        android:name=".app.settings.SettingsActivity"
+        android:label="@string/title_activity_settings"
```
Comment (teamagam5) [inline app/src/main/AndroidManifest.xml]: rename to convention (or rename others)

Comment (teamagam5) [reply app/src/main/AndroidManifest.xml]: `activity_settings_title`

File: app/src/main/java/com/teamagam/gimelgimel/app/mainActivity/view/MainOptionsMenu.java [hunk 2]
```diff
@@ -28,21 +28,21 @@
...
+  private final Navigator mNavigator;
+  private final MenuInflater mMenuInflater;
   private final Context mContext;
-  private final SharedPreferences mDefaultSharedPreferences;
+  private final SharedPreferences mSharedPreferences;
 
-  public MainOptionsMenu(@Provided PreferencesUtils preferencesUtils,
-      @Provided LayersLocalCacheData layersLocalCacheData,
+  public MainOptionsMenu(@Provided LayersLocalCacheData layersLocalCacheData,
+      Navigator navigator,
```
Comment (teamagam5) [inline app/src/main/java/com/teamagam/gimelgimel/app/mainActivity/view/MainOptionsMenu.java]: Coudn't be provided?

**Review by teamagam5 — APPROVED** [review]
ok looks fine except for the `SettingsActivity` - it is not so clean as you can tell... up to you if you want to re-arrange it a bit.

File: app/src/main/java/com/teamagam/gimelgimel/app/mainActivity/view/MainOptionsMenu.java (hunk 2 above)
Comment (BarAgam) [reply app/src/main/java/com/teamagam/gimelgimel/app/mainActivity/view/MainOptionsMenu.java]: Indeed

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (BarAgam) [reply app/src/main/AndroidManifest.xml]: OKie

**Pull request merged**